### PR TITLE
Documentation write components guide

### PR DESCRIPTION
**Pull request by ArshdeepSingh3-eaton**
Fixes # .

#### Changes proposed in this Pull Request:

-Updated the documentation
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/docs/screens/account-details.md
Comment (ektaghag-eaton) [inline login-workflow/docs/screens/account-details.md:1]: Add image here.

File: login-workflow/docs/components/error-manager.md
Comment (ektaghag-eaton) [inline login-workflow/docs/components/error-manager.md:1]: Add images for 
Error Dialog, Error Box similar to react workflows.

File: login-workflow/docs/components/workflow-card.md
Comment (ektaghag-eaton) [inline login-workflow/docs/components/workflow-card.md:1]: The image is missing.

Comment (ektaghag-eaton) [inline login-workflow/docs/components/workflow-card.md:1]: Remove section `workflowCardFinishState` https://github.com/etn-ccis/blui-react-native-workflows/issues/365.

**Review by ektaghag-eaton — COMMENTED** [review]
Remove TODO comment - https://github.com/etn-ccis/blui-react-native-workflows/blob/release/R31/login-workflow/docs/screens/create-password.md?plain=1#L5

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts.

**Review by ektaghag-eaton — APPROVED** [review]

**Pull request merged**